blueprints/app/files/.eslintrc.js

```diff
@@ -2,12 +2,18 @@
 
 module.exports = {
   root: true,
-  parser: '<%= typescript ? '@typescript-eslint/parser' : 'babel-eslint' %>',
+  parser: '<%= typescript ? '@typescript-eslint/parser' : '@babel/eslint-parser' %>',
   parserOptions: {
     ecmaVersion: 2018,<% if (!typescript) { %>
     sourceType: 'module',
     ecmaFeatures: {
       legacyDecorators: true,
+    },
+    requireConfigFile: false,
+    babelOptions: {
+      plugins: [
+        ['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true }],
```

Comment:
**question**

Now that `@babel/plugin-proposal-decorators` is used, I don't think we need the `ecmaFeatures` key (along with `legacyDecorators`) anymore (lines 9-11)?

The first commit (https://github.com/ember-cli/ember-cli/pull/9669/commits/e97c6aded1ee8fba5d8e3894bdf0c0526d05a38b) shows that `ecmaFeatures` had been removed. I wonder if it got accidentally re-introduced by merging `master` (i.e. a rebase didn't happen correctly).